Contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Thanks for being interested in contributing to this project. Please read this before getting involved:
+
+* Please look if someone has already started working on the same issues
+* Please share your intentions in [our mailing list](https://groups.google.com/d/forum/openwisp) and open issues using the [issue tracker](https://github.com/openwisp/openwisp-config/issues).  
+* Pay attention to following [PEP8, Style Guide for Python Code](http://www.python.org/dev/peps/pep-0008/)
+* Fork the code and write code and tests
+* All tests should pass and the test coverage shouldn't decrease
```

Comment:
there's no code coverage tool in this repo because it's not as trivial as doing it with python